src/Azure/Azure.Quantum.Client/JobManagement/ProviderStatusInfo.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+namespace Microsoft.Azure.Quantum
+{
+    using System.Collections.Generic;
```

Comment:
Nit, but most elsewhere you've had `using` blocks outside of `namespace`.

Reply:
Yes, somehow for this project stylecop likes it to be inside the namespace.